public/js/src/game/ui/drawPip.ts

```diff
@@ -1,164 +1,762 @@
+    ctx.beginPath();
+    ctx.moveTo(118.989890, 90.936255);
+    ctx.bezierCurveTo(119.304210, 91.041735, 119.369090, 91.109475, 119.802410, 91.787675);
+    ctx.bezierCurveTo(121.387880, 94.269495, 122.408000, 97.267715, 122.818460, 100.652650);
+    ctx.bezierCurveTo(122.952570, 101.759780, 122.952920, 104.338590, 122.819430, 105.349590);
+    ctx.bezierCurveTo(122.706800, 106.202060, 122.515160, 107.140970, 122.326320, 107.771960);
+    ctx.lineTo(122.187000, 108.237100);
+    ctx.lineTo(121.930550, 108.289600);
+    ctx.bezierCurveTo(121.178000, 108.452940, 120.443930, 108.579590, 120.384560, 108.556660);
+    ctx.bezierCurveTo(120.345760, 108.542460, 120.286860, 108.397910, 120.250450, 108.235970);
+    ctx.bezierCurveTo(119.925730, 106.801330, 118.037060, 104.644610, 115.188270, 102.455360);
+    ctx.bezierCurveTo(112.247360, 100.195280, 108.836670, 98.496875, 106.756570, 98.256765);
+    ctx.bezierCurveTo(105.978510, 98.167165, 105.387610, 97.957585, 105.267410, 97.729075);
+    ctx.bezierCurveTo(105.074040, 97.360785, 105.669280, 96.739035, 107.188930, 95.723645);
+    ctx.bezierCurveTo(110.261110, 93.670755, 114.325170, 91.716045, 116.863790, 91.070105);
+    ctx.bezierCurveTo(117.650790, 90.869705, 118.613070, 90.808665, 118.989640, 90.934275);
+    if (colors) {
+      ctx.fillStyle = colors[4];
+      ctx.fill();
+    }
+    ctx.closePath();
+
+// 2517
+    ctx.beginPath();
+    ctx.moveTo(109.895010, 106.209430);
+    ctx.bezierCurveTo(112.598280, 106.441230, 115.232040, 107.157590, 117.396170, 108.249980);
+    ctx.bezierCurveTo(118.112170, 108.611700, 119.045740, 109.169090, 119.308260, 109.392210);
+    ctx.lineTo(119.481190, 109.538970);
+    ctx.lineTo(119.396290, 110.419390);
+    ctx.bezierCurveTo(119.350890, 110.903580, 119.300590, 111.311450, 119.287820, 111.325810);
+    ctx.bezierCurveTo(119.273920, 111.340010, 118.942990, 111.372010, 118.550930, 111.391810);
+    ctx.bezierCurveTo(117.515810, 111.444310, 116.911730, 111.632420, 115.457470, 112.342470);
+    ctx.bezierCurveTo(112.806010, 113.637470, 109.673070, 116.012570, 107.539510, 118.345080);
+    ctx.bezierCurveTo(106.571780, 119.403040, 106.039650, 120.152220, 105.409360, 121.344050);
+    ctx.bezierCurveTo(104.742760, 122.604370, 104.485370, 122.824760, 104.131060, 122.438620);
+    ctx.bezierCurveTo(103.598470, 121.858260, 102.938220, 119.465550, 102.378670, 116.089090);
+    ctx.bezierCurveTo(101.960580, 113.564930, 101.900420, 112.892630, 101.897820, 110.696790);
+    ctx.bezierCurveTo(101.895820, 108.890120, 101.900820, 108.741240, 102.014960, 108.289900);
+    ctx.bezierCurveTo(102.158430, 107.709460, 102.343840, 107.321480, 102.568270, 107.129110);
+    ctx.bezierCurveTo(102.855560, 106.882850, 104.280190, 106.522360, 105.763100, 106.320830);
+    ctx.bezierCurveTo(106.706170, 106.192410, 108.973620, 106.132430, 109.895770, 106.211460);
+    if (colors) {
+      ctx.fillStyle = colors[0];
+      ctx.fill();
+    }
+    ctx.closePath();
+
+    //2521
```

Comment:
what does this comment mean

Reply:
It's the name of the path in my svg file, is there for my own sanity, but I can change it to something meaningful.